packages/hardhat/contracts/Pools.sol

```diff
@@ -0,0 +1,161 @@
+pragma solidity ^0.8.4;
+//SPDX-License-Identifier: MIT
+
+import "hardhat/console.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "prb-math/contracts/PRBMath.sol";
+import "prb-math/contracts/PRBMathUD60x18.sol";
+
+import "./interfaces/IPools.sol";
+import "./interfaces/Validated.sol";
+
+contract Pools is IPools, Validated {
+    using SafeERC20 for ERC20;
```

Comment:
we don't need any kind of `SafeMath` with Solidity >0.8.0

Reply:
what about `safeTransferFrom`?

Reply:
That's a different thing, what you don't need is to check for overflow and underflow from solidity 0.8, that's basically what SafeMath does